.github/workflows/buildcheck.yml

```diff
@@ -12,6 +12,7 @@ on:
       - 'lib/**'
       - 'lib.src/**'
       - 'src/**'
+      - '.github/workflows/buildcheck.yml'
```

Comment:
Bitte ändere das noch in ` - '.github/workflows/**'`.

Reply:
Kann ich machen, am Ende des Tages macht das aber nichts sicherer. Auch action pinning verhindert nicht, dass prs prinzipiell laufen, sobald man in einem der genannten Verzeichnisse Änderungen durchgeführt werden. Wenn bei einem pr eine Action aktiviert wird, passiert das einfach. Das

```yaml
on:
  pull_request:
```
führt am Ende des Tages immer einen build durch, da ich als potenzieller Angreifer einfach nur in einem der Pfade eine Änderung anpassen muss. Da wäre ein on: push einfacher.

Reply:
Ich habe mal geschaut. Wir sollten was mit [CODEOWNERS](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-codeowners-to-monitor-changes) machen, damit nicht jeder beliebig workflows ändern oder erstellen kann.

Reply:
> Ich habe mal geschaut. Wir sollten was mit [CODEOWNERS](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-codeowners-to-monitor-changes) machen, damit nicht jeder beliebig workflows ändern oder erstellen kann.

Gute Idee. Das hindert wohl jedoch nicht daran, dass eine Action in einem PR ausgeführt wird. Nur dass ein Review dazu nötig ist.

Reply:
> Da wäre ein on: push einfacher.

Wie meinst du das? Wenn ich in eine bereits bestehende PR-branch einen workflow pushe, läuft das doch auf das gleiche raus. Oder?